tests/conformance_suite.postman_collection.json

```diff
@@ -611,72 +611,6 @@
 									},
 									"response": []
 								},
-								{
-									"name": "credentials_issue:credential.@context:invalid",
-									"event": [
-										{
-											"listen": "test",
-											"script": {
-												"exec": [
-													"pm.test(\"status code is 400\", function () {",
-													" pm.response.to.have.status(400);",
-													"});",
-													"",
-													"pm.test(\"response validates against schema\", function() {",
-													" const schemaString = pm.collectionVariables.get(\"responseSchema400\");",
-													" pm.response.to.have.jsonSchema(JSON.parse(schemaString));",
-													"});",
-													"",
-													""
-												],
-												"type": "text/javascript"
-											}
-										},
-										{
-											"listen": "prerequest",
-											"script": {
-												"exec": [
-													"pm.variables.set(\"requestBody\", mutateRequestBody((req) => {",
-													"    // \"credential.@context\" must contain \"https://w3id.org/traceability/v1\"",
```

Comment:
I get the intention of this test, but it will force all conforming implementers to fail to verify "standard"  non trace verifiable credentials, including GS1 credentials:

https://w3c-ccg.github.io/traceability-vocab/#GS1KeyCredential

Therefore this test must be removed.